ReadMe.md

```diff
@@ -1,86 +1,32 @@
-![logo.png](misc/assets/gdext-ferris.png)
-
-# Rust bindings for Godot 4
```

Comment:
did you intentionally replace the entire readme file?

Reply:
Oh no, that was to test out some Markdown. Thanks a lot, great catch!